renderer.php

```diff
@@ -232,6 +232,31 @@ public function object_status_page_intro() {
         return $output;
     }
 
+    /**
+     * Delete test files from files table
+     * @throws dml_exception
+     */
+    public function delete_presignedurl_tests_files() {
+        global $CFG;
+        $filestorage = get_file_storage();
+        $fixturespath = $CFG->dirroot . '/admin/tool/objectfs/tests/fixtures/';
+        $fixturesfiles = glob($fixturespath.'*');
+        foreach ($fixturesfiles as $fixturesfile) {
```

Comment:
This should not delete all the fixtures, it should delete all the files in the file api file area

Reply:
https://docs.moodle.org/dev/File_API#List_area_files

eg

```
$fs = get_file_storage();
$files = $fs->get_area_files($contextid, 'mod_assignment', 'submission', $submission->id);
foreach ($files as $f) {
    $f->delete();
}
```